mxnet/inference/buildspec.yml

```diff
@@ -28,19 +28,6 @@ context:
       target: deep_learning_container.py
```

Comment:
what are these buildspec changes? can you revert it?

Reply:
I don't want to build all images and just one image is needed, so build changes are there. Those build changes are to be reverted before merge